models/commons/src/price/price-draft/transformers.ts

```diff
@@ -31,6 +32,12 @@ const transformers = {
       'discounted',
       'custom',
     ],
+    replaceFields: ({ fields }) => ({
+      ...fields,
+      value: {
+        centPrecision: fields.value as TMoneyGraphql,
```

Comment:
Here's the shim to "fake" a `BaseMoneyInput` model. We can always create it down the line if needed.